data/json/recipes/recipe_vehicle.json

```diff
@@ -1186,5 +1186,33 @@
     "proficiencies": [ { "proficiency": "prof_elec_soldering" } ],
     "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "SCREW_FINE", "level": 1 } ],
     "components": [ [ [ "e_scrap", 4 ] ], [ [ "cable", 30 ] ] ]
+  },
+  {
+    "type": "recipe",
```

Comment:
Please put this next to the existing controls recipe instead of appending it to the end of the file.

Reply:
Moved as requested.